jekyll-kuma-plugins/lib/jekyll/kuma-plugins/liquid/tags/policyyaml.rb

```diff
@@ -52,16 +57,18 @@ def process_array(array, remove_suffix, rename_suffix)
 
           def render(context)
             content = super
-            return "" unless content != ""
+            return "" if content == ""
+            has_raw = @body.nodelist.first { |x| x.has?("tag_name") and x.tag_name == "raw"}
```

Comment:
Please make this in a separate PR next time :pray:

Reply:
I can make it a separate PR if you want.